node/middlewares/indexRoutes.ts

```diff
@@ -16,11 +16,22 @@ const index = async (ctx: Context, next: () => Promise<void>) => {
     productsWithoutSKU,
   }: IndexRoutesEvent = body
 
-  const dataFile = await vbase.getJSON<{ indexBucket: string }>(BUCKET, FILE, true)
+  const { data: dataFile, headers: {etag} } = await vbase.getRawJSON<IndexRoutesEvent>(BUCKET, FILE, true)
```

Comment:
Could this response be a cached one since `read-after-write`  is not guaranteed? if it is the case we should add the `no-cache` header

Reply:
Caching defeats the purpose of using this file to store the `from`, since we save it at the end of the function.
I've added the `Cache-Contol: no-cache` header to the client.